### PR TITLE
home, choose, fonts, hover states

### PR DESCRIPTION
**Pull request by evanmac123**

### CODE REVIEW

**Comment by evanmac123** [comment]
@mikehwagz give me some crit and then we can merge. I still need help selecting elements using config.this.el

File: src/scss/base/index.scss [hunk 1]
```diff
@@ -21,11 +21,54 @@ html, body {
   margin: 0;
   background-color: $dark-grey;
   color: $white;
+  font-family: 'Chivo', sans-serif;
+
+}
+
+white{
```
Comment (mikehwagz) [inline src/scss/base/index.scss]: are all of these color rules meant to be targeting classes? you need to add the `.` before them. also, would you mind adding spaces between selectors and brackets?

File: src/scss/layout/_choose.scss [hunk 2]
```diff
@@ -0,0 +1,52 @@
+.choose__container {
+    @include fullscreen;
+    background-image: url("/assets/images/brick.png");
```
Comment (mikehwagz) [inline src/scss/layout/_choose.scss]: i'm noticing inconsistency with single and double quotes - can we stick to single quotes?

File: src/scss/layout/_choose.scss [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    img{
+      width:100%;
+    }
+
+    .choose__overlay {
+        background-color: #1b1b1b;
+        //mix-blend-mode: multiply;
+        opacity: 0.98;
+        @include fullscreen;
+        z-index: -10;
```
Comment (mikehwagz) [inline src/scss/layout/_choose.scss]: there should be no reason you need z-index of negative 10 here. position absolute will push it above the other content, so if you just make the other content position relative and do z-index of 1 it should go on top of the overlay.

File: src/scss/layout/_choose.scss [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    }
+
+    .choose__body {
+        margin-top: 30px;
+    }
+
+    .choose__person_name{
+      position:relative;
```
Comment (mikehwagz) [inline src/scss/layout/_choose.scss]: can you also make sure you have spaces between colons?

File: src/scss/main.scss [hunk 5]
```diff
@@ -3,11 +3,6 @@
 @import './module/index';
 @import './state/index';
 
-@mixin hidden {
-  opacity: 0;
-  visibility: hidden;
-}
-
```
Comment (mikehwagz) [inline src/scss/main.scss]: why did you delete this mixin?

File: src/templates/home.mst [hunk 6]
```diff
@@ -1 +1,20 @@
...
+<div class="home__main_title"> RECOVERY</div><br>
+<div class="home__main_title"> ROAD</div>
+
+
+
+<h3> Stories of the Boston Opioid Crisis</h3>
+
+<a href="/choose"><button>BEGIN</button></a>
+</div>
+</div>
```
Comment (mikehwagz) [inline src/templates/home.mst]: this is probably just a weird github thing, but just want to make sure this is indented correctly

File: src/scss/main.scss (hunk 5 above)
Comment (evanmac123) [inline src/scss/main.scss]: it is in the mixins folder already

Comment (mikehwagz) [inline src/scss/main.scss]: ah ok cool
